e-commerse/src/Components/Atoms/InputField.tsx

```diff
@@ -0,0 +1,44 @@
+interface propsType {
+  type: string;
```

Comment:
You can use this types by extending the html element as Bharat shared in last call